### ISSUE TITLE
Removing the real device doesn't stop the virtual device.

### ISSUE THREAD

**Issue by KiNGKiMO** [csutorasa/XOutput#128]
Hello,
If you unplug the real gamepad, it disappears from DirectInput list, but it doesn't stop game controllers assigned to removed device.

**Comment by csutorasa** [comment]
This is true, but is it causing any issue? I can be useful when you unplug and plug in the controller again, then you do not need to start it again. While if you disconnect, the device is doing nothing.

The main issue is when you have keyboard and gamepad assigned to one controller, then maybe you can use it even if the gamepad is disconnected.

I know there was a bug, that affected the unplugged controllers (#110) but it is solved now.

**Comment by KiNGKiMO** [comment]
If this is intentional, then it's ok...
I just thought it makes more sense If the real device is removed then the virtual is removed as well so no confusions.
